content/100-getting-started/02-setup-prisma/100-start-from-scratch/110-relational-databases/index.mdx

```diff
@@ -123,8 +123,10 @@ Next, set up your Prisma project by creating your [Prisma schema](/concepts/comp
 npx prisma init
 ```
 
-This command creates a new directory called `prisma` which contains a file named `schema.prisma` and a `.env` file in the root of the project. `schema.prisma` contains the Prisma schema with your database connection and the Prisma Client generator.
-`.env` is a [dotenv](https://github.com/motdotla/dotenv) file for defining environment variables (used for your database connection).
+This command does two things:
+
+- creates a new directory called `prisma` that contains a file called `schema.prisma`, which contains the Prisma schema with your database connection variable and schema models
+- creates the `.env` file in the root directory of the project, which is a [dotenv](https://github.com/motdotla/dotenv) file used for defining environment variables (such as your database connection)
```

Comment:
We probably have a link about what the `.env` file does in Prisma context that could be linked here.

Reply:
Added a link to https://www.prisma.io/docs/guides/development-environment/environment-variables#using-env-files